package.json

```diff
@@ -36,10 +36,12 @@
   "author": "HomeOffice",
   "dependencies": {
     "browserify": "^12.0.1",
-    "hof-bootstrap": "^6.0.0",
+    "hof-bootstrap": "^9.3.1",
```

Comment:
Looking at the controllers for this app (there basically are none) this should be ok.